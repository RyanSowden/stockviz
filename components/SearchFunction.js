import React, {useState} from "react"


const SearchFunction = (props) => {
    const current_price = props.current_stock
    const adjusted_price = props.adjusted_stock
    return(
	<div>
        <h1>{current_price}</h1>
        <h1>{adjusted_price}</h1>
	</div>
    )
}

export default SearchFunction
