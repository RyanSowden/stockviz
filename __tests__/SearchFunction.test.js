import React from "react";
import Enzyme from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import renderer from 'react-test-renderer';
import {shallow, mount} from 'enzyme';
import SearchFunction from '../components/SearchFunction'



Enzyme.configure({adapter: new Adapter()});

describe("SearchFunction", () => {
    it("should render properly", () => {
        const app = renderer.create(<SearchFunction />);
        expect(app).toMatchSnapshot();
    });

})
