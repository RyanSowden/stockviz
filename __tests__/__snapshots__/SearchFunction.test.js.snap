// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchFunction should render properly 1`] = `<h1 />`;
